data/base-healingherbs.yaml

```diff
@@ -393,33 +395,130 @@ herb_stocks:
       size: 6
       stackable: true
       room: 8666
-      price: 500
-      quantity: 7
```

Comment:
Did you mean to remove these two lines?

Reply:
Nope. Must have accidently deleted them when I was adding ratha.